homeassistant/components/todo/const.py

```diff
@@ -4,6 +4,11 @@
 
 DOMAIN = "todo"
 
+CONF_DUE = "due"
+CONF_DUE_DATE = "due_date"
+CONF_DUE_DATE_TIME = "due_date_time"
+CONF_DESCRIPTION = "description"
```

Comment:
These aren't really configuration, I think. It's more attributes.
```suggestion
ATTR_DUE = "due"
ATTR_DUE_DATE = "due_date"
ATTR_DUE_DATE_TIME = "due_date_time"
ATTR_DESCRIPTION = "description"
```